tests/dataset_test.py

```diff
@@ -284,6 +284,7 @@ def test_coords_update_from_sequence_of_tuples_adds_items():
 
 
 def test_coords_update_from_iterable_of_tuples_adds_items():
+
```

Comment:
Strange unrelated change?

Reply:
This showed up in a lot of PRs. It comes from the update of our yapf version.